e2e-tests/tests/test_api.py

```diff
@@ -8,6 +8,12 @@
 import pytest
 
 
+# Because there are so few crashes for these product, instead of
+# looking for crashes from the *active* versions, we'll for any
+# any recent crashes for these.
```

Comment:
This last clause has two "any" and is missing a word. Maybe it should be "we'll look for any recent crashes"?